scripts/checks/compareGasReports.js

```diff
@@ -0,0 +1,219 @@
+            delta: current.avg - previous.avg,
+            ratio: 100 * (current.avg - previous.avg) / (previous.avg + BASE_TX_COST),
+          },
+        })))
+      .filter(rows => rows.length);
+  }
+}
+
+// Display
+function center (text, length) {
+  return text.padStart((text.length + length) / 2).padEnd(length);
+}
+
+function plusSign(num) {
+  return num > 0 ? '+' : '';
+}
+
+function formatCmpBash (rows) {
+  const contractLength = Math.max(...rows.map(x => Math.max(...x.map(({ contract }) => contract.length))));
+  const methodLength = Math.max(...rows.map(x => Math.max(...x.map(({ method }) => method.length))));
+
+  const COLS = [
+    { txt: '', length: 0 },
+    { txt: 'Contract', length: contractLength },
+    { txt: 'Method', length: methodLength },
+    { txt: 'Min', length: 30 },
+    { txt: 'Max', length: 30 },
+    { txt: 'Avg', length: 30 },
+    { txt: '', length: 0 },
+  ];
+  const HEADER = COLS.map(entry => chalk.bold(center(entry.txt, entry.length || 0))).join(' | ').trim();
+  const SEPARATOR = COLS.map(({ length }) => length > 0 ? '-'.repeat(length + 2) : '').join('|').trim();
+
+  return [
+    '',
+    HEADER,
+    ...rows.map(methods => methods.map(entry => [
+      '',
+      chalk.grey(entry.contract.padEnd(contractLength)),
+      entry.method.padEnd(methodLength),
+      /* eslint-disable max-len */
+      chalk[entry.min.delta > 0 ? 'red' : entry.min.delta < 0 ? 'green' : 'reset']((isNaN(entry.min.total) ? '-' : entry.min.total.toString()).padStart(8)),
+      chalk[entry.min.delta > 0 ? 'red' : entry.min.delta < 0 ? 'green' : 'reset']((isNaN(entry.min.delta) ? '-' : plusSign(entry.min.delta) + entry.min.delta.toString()).padStart(8)),
+      chalk[entry.min.delta > 0 ? 'red' : entry.min.delta < 0 ? 'green' : 'reset']((isNaN(entry.min.ratio) ? '-' : plusSign(entry.min.ratio) + entry.min.ratio.toFixed(2) + '%').padStart(8)),
+      chalk[entry.max.delta > 0 ? 'red' : entry.max.delta < 0 ? 'green' : 'reset']((isNaN(entry.max.total) ? '-' : entry.max.total.toString()).padStart(8)),
+      chalk[entry.max.delta > 0 ? 'red' : entry.max.delta < 0 ? 'green' : 'reset']((isNaN(entry.max.delta) ? '-' : plusSign(entry.max.delta) + entry.max.delta.toString()).padStart(8)),
+      chalk[entry.max.delta > 0 ? 'red' : entry.max.delta < 0 ? 'green' : 'reset']((isNaN(entry.max.ratio) ? '-' : plusSign(entry.max.ratio) + entry.max.ratio.toFixed(2) + '%').padStart(8)),
+      chalk[entry.avg.delta > 0 ? 'red' : entry.avg.delta < 0 ? 'green' : 'reset']((isNaN(entry.avg.total) ? '-' : entry.avg.total.toString()).padStart(8)),
+      chalk[entry.avg.delta > 0 ? 'red' : entry.avg.delta < 0 ? 'green' : 'reset']((isNaN(entry.avg.delta) ? '-' : plusSign(entry.avg.delta) + entry.avg.delta.toString()).padStart(8)),
+      chalk[entry.avg.delta > 0 ? 'red' : entry.avg.delta < 0 ? 'green' : 'reset']((isNaN(entry.avg.ratio) ? '-' : plusSign(entry.avg.ratio) + entry.avg.ratio.toFixed(2) + '%').padStart(8)),
```

Comment:
This is calling for a helper like `formatCells(entry.min)`. The same recommendation for the markdown formatter.

```js
const formatCells = (entry) => {
  const color = entry.delta > 0 'red' : entry.delta < 0 ? 'green' : 'reset';
  return [
    chalk[color](...),
    chalk[color](...),
    ...
}
```

```suggestion
      ...formatCells(entry.min),
      ...formatCells(entry.max),
      ...formatCells(entry.avg),
```